extensions/wikia/Search/classes/Result.php

```diff
@@ -351,16 +351,16 @@ private static function replaceUnusualEscapesCallback( $matches ) {
 	 * @param $wordLimit
 	 * @return mixed|string
 	 */
-	public function limitTextLength( $textAsString, $wordLimit ) {
+	public static function limitTextLength( $textAsString, $wordLimit ) {
 		if ( $wordLimit !== null ) {
 			$wordsExploded = explode( ' ', $textAsString );
 			$textAsString = implode( ' ', array_slice( $wordsExploded, 0, $wordLimit ) );
 			if ( count( $wordsExploded ) > $wordLimit ) {
-				$textAsString = $this->fixSnippeting( $textAsString, true );
+				$textAsString = static::fixSnippeting( $textAsString, true );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$textAsString". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)